web/src/main/resources/META-INF/resources/webjars/common/webcert/utkast/utkastHeader/wcUtkastHeader/wcUtkastHeader.directive.js

```diff
@@ -64,8 +67,108 @@ angular.module('common').directive('wcUtkastHeader',
               }
             }
 
+            function discardModal() {
+              if(UserModel.user.origin === 'DJUPINTEGRATION' && $scope.utkastViewState.draftModel.content.name ==='dbModel') {
+                var patient = $scope.utkastViewState.intygModel.grundData.patient;
+                var dialogModel = {
+                  acceptprogressdone: false,
+                  errormessageid: 'Error',
+                  showerror: false,
+                  info: messageService.getProperty('db.label.info',
+                      {fornamn: patient.fornamn,
+                        efternamn: patient.efternamn,
+                        personnummer: patient.personId}),
+                  checkboxCheck: function() {
```

Comment:
I have no idea what this function is about, instead of specifying what it does it would probably be better if it explained what it's for. The comments also doesn't add any additional information

Reply:
Fixed!